docs/wiki_ja/deployment.md

```
@@ -1,297 +1,96 @@
-    steps:
-      - name: Deploy to GitHub Pages
-        id: deployment
-        uses: actions/deploy-pages@v2
-```
-
-## デプロイメントフロー図
-
-以下の図は、コードがリポジトリからウェブサイトにデプロイされるまでの流れを示しています：
+## デプロイフロー
 
 ```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)